src/hiero_sdk_python/tokens/fee_assessment_method.py

```
@@ -2,7 +2,17 @@
 
 class FeeAssessmentMethod(Enum):
     """
-    Represents the fee assessment method for custom fees.
+    Fee assessment method for custom token fees:
+
+      • INCLUSIVE – Fee is deducted from the transferred amount.
```

Ruff (0.14.10)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)